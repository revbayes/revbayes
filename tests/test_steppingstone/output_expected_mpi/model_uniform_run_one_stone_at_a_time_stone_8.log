Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-26650.52	-26649.18	-1.345983	0.9750844	0.129484	0.0114031	0.0474697	0.0935272	0.0658321	0.0496263	0.358445	0.120119	0.0613609	0.237878	0.0907795	0.0994452	0.186458	0.0587717	0.0627727	0.153965	0.0415577	0.279718	0.162017	0.248912	0.162155	0.123703	0.168022	0.020078	0.0970038	0.011277	0.0788618	0.0658689	0.0487606	0.106201	0.069564	0.0167169	0.181427	0.135193	0.191747	0.279345	0.27995	0.0199007	0.0184013	0.0329143	0.0238875	0.0532459	0.0553578	0.013979	0.137052	0.238697	0.152537	0.418698	0.0390373	0.186945	0.443623	0.116977	0.252455	0.7518985	4.799125
10	-30250.31	-30243.51	-6.793917	1.069211	0.268998	0.00866716	0.0302124	0.152032	0.0497011	0.0438801	0.323542	0.0202899	0.0799679	0.164185	0.10844	0.048163	0.210761	0.108273	0.0338967	0.273519	0.00668128	0.477595	0.080661	0.0598693	0.146274	0.257125	0.456174	0.00942788	0.160999	0.0210599	0.0775491	0.0356302	0.122563	0.0574399	0.0413847	0.0794384	0.202577	0.0982195	0.0488242	0.205358	0.381628	0.0330912	0.0577019	0.0528196	0.0227227	0.0498117	0.185504	0.00778112	0.303786	0.322087	0.189699	0.167122	0.00952447	0.354841	0.420968	0.15044	0.0737513	0.7941156	5.352658
