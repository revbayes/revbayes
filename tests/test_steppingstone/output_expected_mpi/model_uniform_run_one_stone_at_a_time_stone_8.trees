Iteration	Posterior	Likelihood	Prior	psi
0	-27249.6	-27247.45	-2.156013	((Macaca_mulatta[&index=13]:0.202086,(Aotus_trivirgatus[&index=1]:0.102324,((Pan_paniscus[&index=17]:0.059746,(Tarsius_syrichta[&index=21]:0.139759,(Chlorocebus_aethiops[&index=5]:0.030257,(Saimiri_sciureus[&index=20]:0.078093,((Callicebus_donacophilus[&index=2]:0.151081,Cebus_albifrons[&index=3]:0.100839)[&index=26]:0.038797,((Galago_senegalensis[&index=8]:0.022995,(Loris_tardigradus[&index=12]:0.058210,(Lemur_catta[&index=10]:0.029187,(Cheirogaleus_major[&index=4]:0.077039,(Nycticebus_coucang[&index=15]:0.062553,((((Hylobates_lar[&index=9]:0.097827,Microcebus_murinus[&index=14]:0.045373)[&index=29]:0.376758,(Lepilemur_hubbardorum[&index=11]:0.048892,Perodicticus_potto[&index=18]:0.043006)[&index=41]:0.090338)[&index=30]:0.080265,Otolemur_crassicaudatus[&index=16]:0.314947)[&index=36]:0.659973,Daubentonia_madagascariensis[&index=7]:0.518640)[&index=32]:0.022302)[&index=37]:0.091701)[&index=31]:0.006321)[&index=38]:0.216272)[&index=39]:0.032059)[&index=25]:0.041192,Propithecus_coquereli[&index=19]:0.016448)[&index=42]:0.154921)[&index=34]:0.077140)[&index=28]:0.050504)[&index=24]:0.001981)[&index=33]:0.084445)[&index=40]:0.124206,Varecia_variegata_variegata[&index=22]:0.284743)[&index=27]:0.035212)[&index=43]:0.052102)[&index=35]:0.023272,Galeopterus_variegatus[&index=23]:0.107538,Colobus_guereza[&index=6]:0.015952)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
10	-28394.44	-28387.67	-6.774116	(Colobus_guereza[&index=6]:0.009614,Galeopterus_variegatus[&index=23]:0.248875,(Callicebus_donacophilus[&index=2]:0.299494,(Nycticebus_coucang[&index=15]:0.068548,((Lemur_catta[&index=10]:0.258487,((Hylobates_lar[&index=9]:0.148362,Perodicticus_potto[&index=18]:0.050189)[&index=25]:0.107424,(((Daubentonia_madagascariensis[&index=7]:0.145930,Microcebus_murinus[&index=14]:0.089414)[&index=41]:0.057559,((Saimiri_sciureus[&index=20]:0.093954,Otolemur_crassicaudatus[&index=16]:0.259655)[&index=32]:0.022926,((Galago_senegalensis[&index=8]:0.072989,Loris_tardigradus[&index=12]:0.139753)[&index=39]:0.014031,((Propithecus_coquereli[&index=19]:0.036205,Lepilemur_hubbardorum[&index=11]:0.164276)[&index=36]:0.770547,Cebus_albifrons[&index=3]:0.059970)[&index=30]:0.169269)[&index=42]:0.155487)[&index=29]:0.240219)[&index=37]:0.061983,(Aotus_trivirgatus[&index=1]:0.042437,Macaca_mulatta[&index=13]:0.120400)[&index=43]:0.207136)[&index=27]:0.090967)[&index=28]:0.034248)[&index=26]:0.045129,(Tarsius_syrichta[&index=21]:0.059138,(((Chlorocebus_aethiops[&index=5]:0.018911,Cheirogaleus_major[&index=4]:0.053310)[&index=31]:0.022679,Pan_paniscus[&index=17]:0.056568)[&index=24]:0.002756,Varecia_variegata_variegata[&index=22]:0.208302)[&index=40]:0.128690)[&index=35]:0.126367)[&index=38]:0.131605)[&index=33]:0.166343)[&index=34]:0.045570)[&index=44];
