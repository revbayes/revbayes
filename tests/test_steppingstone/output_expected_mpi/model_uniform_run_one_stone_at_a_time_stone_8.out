state	power	likelihood
4	0.000541923	-27155.8
5	0.000541923	-27179.3
6	0.000541923	-27147.2
7	0.000541923	-29331.5
8	0.000541923	-28834.3
9	0.000541923	-29235.5
10	0.000541923	-30243.5
